packages/account/src/providers/provider.ts

```diff
@@ -1131,40 +1123,18 @@ Supported fuel-core version: ${supportedVersion}.`
    */
   async getTransactionCost(
     transactionRequestLike: TransactionRequestLike,
-    { resourcesOwner, signatureCallback, quantitiesToContract = [] }: TransactionCostParams = {}
-  ): Promise<TransactionCost> {
+    { signatureCallback }: TransactionCostParams = {}
+  ): Promise<Omit<TransactionCost, 'requiredQuantities'>> {
```

Comment:
Given these changes are already breaking, why not just remove the `requiredQuantities` from the `TransactionCost` type?

Reply:
The `requiredQuantities` is required for the `TransactionCost` result in the `Account`. I've chosen to share this type between the `Provider.getTransactionCost` and `Account.getTransactionCost` methods, hence the omission here.